.github/workflows/test.yml

```diff
@@ -5,10 +5,14 @@ on:
     branches:
       - master
       - develop
+  pull_request:
```

Comment:
So if I get it right that's temporary, just to run the tests on this pull request

Reply:
Exactly, I will revert it now